modules/dxp/apps/search-experiences/search-experiences-service/src/main/java/com/liferay/search/experiences/internal/info/collection/provider/SXPBlueprintInfoCollectionProvider.java

```diff
@@ -114,7 +115,7 @@ public InfoPage<AssetEntry> getCollectionInfoPage(
 				).from(
 					pagination.getStart()
 				).size(
-					pagination.getEnd()
+					pagination.getEnd() - pagination.getStart()
```

Comment:
Good points: I've moved them into searchContext setters instead where we have the same end-start methods.